### ISSUE TITLE
Get ComboBox Selected Value

### ISSUE THREAD

**Issue by magierska** [2gis/Winium.Desktop#248]
How should I get ComboBox selected value using Winium.Desktop? 
Using UISpy I can see, that only attribute, which can show me selected value is Selection in SelectionPatterns.
![image](https://user-images.githubusercontent.com/28708104/42559037-372236f4-84f3-11e8-88e1-7326082f6267.png)

Although this code does not work for me:
`var selected = webElement.GetAttribute("Selection");` 
It returns null.
Basically, how to get attribute, which is not in AutomationElement, but in some ControlPatterns class?

**Comment by voloxastik** [comment]
Hi, try using Visual UI Automation Verify 2.0
[Home Page](https://archive.codeplex.com/?p=uiautomationverify)

Python
``` 
# go to element ComboLBox and click mouse (open ComboLBox)
ActionChains(driver).move_to_element(listorder33).click(listorder33).perform()
# find all element in ComboLBox
path_list_action = list.find_element_by_class_name("ComboLBox").find_elements_by_id('')
print "for list_action -> ", len(path_list_action)  #  print list length

for list_action in path_list_action:
 action_name = list_action.get_attribute('Name')

 if action is not False:
   path_action = list.find_element_by_name("Editing Panel").find_element_by_name("Editing Control").find_element_by_class_name("ComboLBox").find_element_by_name(action)
   print("click action")
   ActionChains(driver).move_to_element(path_action).click(path_action).perform()

```
